pkg/system/phase1_verifying.go

```diff
@@ -171,6 +173,18 @@ func (r *Reconciler) CheckSystemCR() error {
 			fmt.Sprintf("Autoscaler %s missing prometheusNamespace property ", r.NooBaa.Spec.Autoscaler.AutoscalerType))
 	}
 
+	// Rejecting NooBaa CR if 'BucketLoggingStorageClass' is not provided for 'guaranteed' logging and
+	// also the operator is not running in an ODF environment.
+	sc := &storagev1.StorageClass{
+		TypeMeta:   metav1.TypeMeta{Kind: "StorageClass"},
+		ObjectMeta: metav1.ObjectMeta{Name: "ocs-storagecluster-cephfs"},
+	}
+	if !util.KubeCheck(sc) && (r.NooBaa.Spec.BucketLogging.LoggingType == nbv1.BucketLoggingTypeGuaranteed) &&
```

Comment:
Any reason for the brackets? 
```suggestion
	if !util.KubeCheck(sc) && r.NooBaa.Spec.BucketLogging.LoggingType == nbv1.BucketLoggingTypeGuaranteed &&
```

Reply:
To me it make more sense to first check if we have Guaranteedlogging or not and then check for storage classes - 
if (r.NooBaa.Spec.BucketLogging.LoggingType == nbv1.BucketLoggingTypeGuaranteed)  {

        sc := &storagev1.StorageClass{
		TypeMeta:   metav1.TypeMeta{Kind: "StorageClass"},
		ObjectMeta: metav1.ObjectMeta{Name: "ocs-storagecluster-cephfs"},
	}
         if !util.KubeCheck(sc) && 
                  r.NooBaa.Spec.BucketLogging.BucketLoggingStorageClass == nil

-------
-------
}

Point is, why even  bother about storageclass if we don't need at all at first place.

Reply:
Yes Jacky, we can remove the brackets. Thanks Ashish, I am also thinking of the same, will update it.